WooCommerce/Classes/ViewRelated/ReusableViews/ButtonTableViewCell.swift

```diff
@@ -62,7 +62,7 @@ private extension ButtonTableViewCell {
     }
 }
 
-private extension ButtonTableViewCell {
+extension ButtonTableViewCell {
```

Comment:
Q: Is there a reason why this is no more `private`?

Reply:
Before this PR, we only need to customize a `ButtonTableViewCell`'s bottom spacing to 0pt. However, in this PR we could be showing the default bottom spacing (20pt) or 0pt in `OrderDetailsDataSource.configureMarkCompleteButton`. I thought it'd be better using the same default value from the cell in case we change it in the future. Lemme know if you have other thoughts!